definitions/checks/candlepin/cp_env_content/verify_bad_content_definitions.rb

```diff
@@ -0,0 +1,21 @@
+module Checks::Candlepin
+  module CpEnvContent
```

Comment:
`Cp` can be dropped here? since we are already scoping this checks with `Candlepin`

Reply:
@swapab,
There is table having name cp_env_content thats why I mentioned it here.
Please confirm then I will go and remove it.

Reply:
Oh okay makes sense. Can keep it as is.

Reply:
Do we even need a separate namespace for this?

Reply:
@iNecas,
I wanted to give some context of table in candlepin DB + action for that table.
So, added namespace for procedure to avoid long class name.
That's why in checks also I have placed the namespace.
What I can do is remove namespace from check and modify name of check to get a context of table.

Please let me know your suggestions on it so that I can maintain context of `table + action` on it.